databases.go

```diff
@@ -271,6 +291,55 @@ type DatabaseDB struct {
 	Name string `json:"name"`
 }
 
+// DatabaseTopic represents a Kafka topic
+type DatabaseTopic struct {
+	Name              string       `json:"name"`
+	PartitionCount    *uint32      `json:"partition_count,omitempty"`
+	ReplicationFactor *uint32      `json:"replication_factor,omitempty"`
+	Config            *TopicConfig `json:"config,omitempty"`
```

Comment:
I'm seeing a `state` attribute in the response. Should that be added here?

```suggestion
	State             string       `json:"state,omitempty"`
	Config            *TopicConfig `json:"config,omitempty"`
```

Reply:
Agree, added.